src/core/sys/darwin/sys/event.d

```diff
@@ -86,29 +122,70 @@ enum
 enum
 {
     /*
-     * data/hint flags/masks for EVFILT_USER, shared with userspace
+     * Filter specific flags for EVFILT_READ
      *
+     * The default behavior for EVFILT_READ is to make the "read" determination
+     * relative to the current file descriptor read pointer.
+     *
+     * The EV_POLL flag indicates the determination should be made via poll(2)
+     * semantics. These semantics dictate always returning true for regular files,
+     * regardless of the amount of unread data in the file.
+     *
+     * On input, EV_OOBAND specifies that filter should actively return in the
+     * presence of OOB on the descriptor. It implies that filter will return
+     * if there is OOB data available to read OR when any other condition
+     * for the read are met (for example number of bytes regular data becomes >=
+     * low-watermark).
+     * If EV_OOBAND is not set on input, it implies that the filter should not actively
+     * return for out of band data on the descriptor. The filter will then only return
+     * when some other condition for read is met (ex: when number of regular data bytes
+     * >=low-watermark OR when socket can't receive more data (SS_CANTRCVMORE)).
+     *
+     * On output, EV_OOBAND indicates the presence of OOB data on the descriptor.
+     * If it was not specified as an input parameter, then the data count is the
+     * number of bytes before the current OOB marker, else data count is the number
+     * of bytes beyond OOB marker.
+     */
+    EV_POLL   = EV_FLAG0,
+    EV_OOBAND = EV_FLAG1,
+}
+
+enum
+{
+    /*
+     * data/hint fflags for EVFILT_USER, shared with userspace
+     */
+
+     /*
+     * On input, NOTE_TRIGGER causes the event to be triggered for output.
+     */
+    NOTE_TRIGGER = 0x01000000,
+
+    /*
      * On input, the top two bits of fflags specifies how the lower twenty four
      * bits should be applied to the stored value of fflags.
      *
      * On output, the top two bits will always be set to NOTE_FFNOP and the
      * remaining twenty four bits will contain the stored fflags value.
      */
     NOTE_FFNOP      = 0x00000000, /* ignore input fflags */
-    NOTE_FFAND      = 0x40000000, /* AND fflags */
-    NOTE_FFOR       = 0x80000000, /* OR fflags */
+    NOTE_FFAND      = 0x40000000, /* and fflags */
+    NOTE_FFOR       = 0x80000000, /* or fflags */
     NOTE_FFCOPY     = 0xc0000000, /* copy fflags */
     NOTE_FFCTRLMASK = 0xc0000000, /* masks for operations */
     NOTE_FFLAGSMASK = 0x00ffffff,
 
-    NOTE_TRIGGER    = 0x01000000, /* Cause the event to be
```

Comment:
@DmitryOlshansky Do you want me to move it back down (since your review is marked as `changes requested`)?